man/journald.conf.xml

```diff
@@ -78,6 +78,11 @@
         the kernel log buffer, or a syslog socket will still work). Defaults to <literal>auto</literal> in
         the default journal namespace, and <literal>persistent</literal> in all others.</para>
 
+        <para>Note that journald will initially uses volatile storage until a call to 
```

Comment:
nit: /will initially uses/will initially use/

also ad a comma after `storage`

Reply:
ah that was kinda obvious ^^. I thought about that comma, but when to place a comma in english is something I never really understood xD